### PR TITLE
Feature: add computer basics lesson and other changes

### PR DESCRIPTION
**Pull request by 01zulfi**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behaviour
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes

Complete the following checkbox ONLY IF it is applicable to your PR. You can complete it later if it is not currently applicable:
-   [ ] I have verified all tests and linters pass against my changes, and/or I have included automated tests where applicable

<hr>

**1. Because:**

1. There's a new computer basics lesson: https://github.com/TheOdinProject/curriculum/pull/24284
2. Changes in course structure: https://github.com/TheOdinProject/curriculum/pull/24295

**2. This PR:**

1. Add seed for computer basics lesson
2. Make changes according to: https://github.com/TheOdinProject/curriculum/pull/24295

**3. Additional Information:**

n/a

### CODE REVIEW

**Comment by ChargrilledChook** [comment]
@01zulfi Rubocop is strict huh :sweat_smile: 

If you like, there's a handy shortcut to run all linters and tests in one command - `bin/test` should work if run from the app root

**Comment by 01zulfi** [comment]
@ChargrilledChook This is great! Thanks for letting me know :rocket:

File: db/fixtures/paths/foundations/seed.rb [hunk 1]
```diff
@@ -35,21 +35,22 @@
...
 course.add_section do |section|
-  section.title = 'Installations'
-  section.description = 'In this section you will configure your development environment and install some software necessary for web development.'
+  section.title = 'Prerequisites'
+  section.description = 'In this section you\'ll acquire the baseline knowledge before getting into programming. Additionally, you\'ll configure your development environment and install some software necessary for web development.'
   section.identifier_uuid = 'a6ff8570-b301-4275-8cb4-1847a8f8ae25'
 
   section.add_lessons(
+    foundation_lessons.fetch('Computer Basics'),
+    foundation_lessons.fetch('How Does the Web Work?'),
```
Comment (KevinMulhern) [inline db/fixtures/paths/foundations/seed.rb:51]: Moving how does the web work into Prerequisites is great idea 🔥 fits perfectly

**Review by KevinMulhern — APPROVED** [review]
Looks great to me @01zulfi just need the curriculum changes merged for final QA.

Screenshot 2022-06-26 at 13 36 53

**Review by KevinMulhern — APPROVED** [review]
🚀

**Pull request merged**